backend/models/Order.js

```diff
@@ -0,0 +1,11 @@
+const Sequelize = require('sequelize');
```

Comment:
Dont leave files that arent in the code path in the codebase...

If a file isnt being used it shouldn't be here